content/beginner/080_scaling/install_kube_ops_view.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Install Kube-ops-view"
+date: 2020-10-01T08:30:11-07:00
+weight: 5
+---
+
+Before starting to learn about the various auto-scaling options for your EKS cluster we are going to install [Kube-ops-view](https://github.com/hjacobs/kube-ops-view) from **[Henning Jacobs](https://github.com/hjacobs)**.\
+Kube-ops-view provides a common operational picture for a Kubernetes cluster that helps with understanding our cluster setup in a visual way.
+
+{{% notice note %}}
+We will deploy kube-ops-view using `Helm` configured in a previous [module](/beginner/060_helm/helm_intro/install/index.html)
+{{% /notice %}}
+
+The following line updates the stable helm repository and then installs kube-ops-view using a LoadBalancer Service type and creating a RBAC (Resource Base Access Control) entry for the read-only service account to read nodes and pods information from the cluster.
+
+```
+helm install kube-ops-view \
+stable/kube-ops-view \
+--set service.type=LoadBalancer \
+--set nodeSelector.intent=control-apps \
```

Comment:
This selector appears to filter out all 3 of my available nodes. When I remove it, things seem fine. Do we need it?

Reply:
Hi, no, we don't need it, we can remove it.